www/js/lib/uiUtil.js

```diff
@@ -402,7 +473,7 @@ define(rqDef, function() {
         }
         // If we are in Config and a real document has been loaded already, expose return link so user can see the result of the change
         // DEV: The Placeholder string below matches the dummy article.html that is loaded before any articles are loaded
-        if (document.getElementById('liConfigureNav').classList.contains('active') &&
+        if (document.getElementById('liConfigureNav').classList.contains('active') && doc &&
```

Comment:
Is this change done on purpose?

Reply:
Yes, this was done on purpose because I found an exception was being thrown on doc.title in certain circumstances before a ZIM was loaded, and it blocked loading of configuration values, I believe when communicating across implementations.